_posts/2015-05-05-the-case-for-asynchronous-feedback.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: The Case For Asynchronous Feedback
+author: Stuart Kent
+tags: feedback, process
+summary: Giving and receiving feedback is vital to professional (and sometimes personal) growth. Despite this, the potential for feedback to be seen as critical or confrontational means it is tempting to delay or avoid delivering it. Kicking the conversation off via an asynchronous communication channel can help.
+
+---
+
+Giving and receiving feedback is vital to professional (and sometimes personal) growth. Despite this, the potential for feedback to be seen as critical or confrontational means it is tempting to delay or avoid delivering it. Kicking the conversation off via an asynchronous communication channel can help.
+
+My team just began trialling [sayat.me](http://sayat.me/)[^1] as a feedback-facilitating service, which prompted me to think more about this topic. The opinions below are all influenced by my own experiences, and as such are unlikely to be free of bias (i.e. YMMV). Discussion, debate, and (naturally) feedback on the perspectives shared below are welcome - hit me up in the comments section.
+
+### Benefits of giving feedback asynchronously
+
+- **The feedback provider has time to craft and double-check their comments.**
+For feedback to be valuable, it should be both objective and actionable. Sequencing and phrasing can greatly influence how suggestions will be interpreted. Conveying your thoughts in person leaves little margin for error, and any mistakes may be hard to rectify on the spot. On the other hand, writing out feedback allows you to carefully check your tone and phrasing before delivery.
+
+- **The feedback recipient has time to reflect before a response is required.**
+Even if your constructive criticism ticks all the right boxes (accurate, neutral, well-phrased, actionable, etc.), the recipient may still be caught off-guard by your insights. That's ok, and totally expected. What's awkward is for the recipient to have to _respond_ to you immediately, before they have had time to digest what you have said. Allowing the recipient time alone to examine your suggestions, think back over their past actions, and place your recommendations in their proper context should ultimately lead to much less defensive, and much more productive, discussions.[^2]
+
+- **Asynchronous communication channels are inherently more impersonal.**
+Using a less-personal method of communication may help negative feedback feel like less of a personal 'attack'. While careful wording is probably a better way of dealing with this issue, every little helps.
```

Comment:
That's cool, didn't know that! Must be the language barrier between us. :smiley_cat: